src/schema/persons.js

```diff
@@ -35,5 +67,126 @@ exports.typeDefs = `
     BOARD_MEMBER
     SUPPORT
   }
+  
+  type Social @beehiveTable(table_name: "socials", pk_column: "social_id") {
+    social_id: ID!
+    url: String
+    type: SocialType!
+  }
+  
+  enum SocialType {
+    LINKED_IN
+    TWITTER
+    FACEBOOK
+    YOUTUBE
+    INSTAGRAM
+    GOOGLE_PLUS
```

Comment:
GOOGLE_PLUS is being shutdown, should we continue to track it?

Reply:
naw